frontend/platoon-console/src/pages/Homepage.jsx

```diff
@@ -1,11 +1,108 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, CardBody, CardTitle, Modal, ModalBody, Button } from 'reactstrap';
 
 const Homepage = () => {
+  const [agendaModalOpen, setAgendaModalOpen] = useState(false);
+  const [monthlyModalOpen, setMonthlyModalOpen] = useState(false);
+  const [feedbackModalOpen, setFeedbackModalOpen] = useState(false);
+
+  const toggleAgendaModal = () => {
+    setAgendaModalOpen(!agendaModalOpen);
+  };
+
+  const toggleMonthlyModal = () => {
+    setMonthlyModalOpen(!monthlyModalOpen);
+  };
+
+  const toggleFeedbackModal = () => {
+    setFeedbackModalOpen(!feedbackModalOpen);
+  };
+
   return (
     <>
-    <h1 className="mainH1">Platoon Console</h1>
+      <h2 className="mainH2">Platoon Console</h2>
+      <div className="card-container" style={{ marginTop: "2rem", display: "flex", flexWrap: "wrap", justifyContent: "center" }}>
+
+        <Card className="consoleCard">
+          <CardBody>
+            <CardTitle style={{marginBottom: "0"}}>
+              <div style={{ display: "flex", alignItems: "center", justifyContent: "center"}}>
+                <h3>Daily Tasks</h3>
+              </div>
+            </CardTitle>
+            <ul className="consoleCardUl">
+              <li><Link to="rollcall/">Attendance Check In</Link></li>
+              <li><Link onClick={toggleFeedbackModal}>Daily Feedback Form</Link></li>
+            </ul>
+          </CardBody>
+        </Card>
+
+        <Card className="consoleCard">
+          <CardBody>
+            <CardTitle style={{marginBottom: "0"}}>
+              <div style={{ display: "flex", alignItems: "center", justifyContent: "center"}}>
+                <h3>Calendar</h3>
+              </div>
+            </CardTitle>
+            <ul className="consoleCardUl">
+              <li><Link onClick={toggleAgendaModal}>Weekly Agenda</Link></li>
+              <li><Link onClick={toggleMonthlyModal}>Current Month</Link></li>
+            </ul>
+          </CardBody>
+        </Card>
+
+        <Card className="consoleCard">
+          <CardBody>
+            <CardTitle style={{marginBottom: "0"}}>
+              <div style={{ display: "flex", alignItems: "center", justifyContent: "center"}}>
+                <h3>GitHub</h3>
+              </div>
+            </CardTitle>
+            <ul className="consoleCardUl">
+              <li><Link to="https://github.com/Code-Platoon-Curriculum" target="_blank" rel="noopener noreferrer">Curriculum</Link></li>
+              <li><Link to="https://github.com/Code-Platoon-Curriculum/whiskey-demos-and-notes" target="_blank" rel="noopener noreferrer">Demos & Notes</Link></li>
+              <li><Link to="https://github.com/Code-Platoon-Assignments/" target="_blank" rel="noopener noreferrer">Assignments</Link></li>
```

Comment:
I know the assignments are usually linked from the readme.  If we want an open or generic link we can put this as an endpoint or keep it static.  This could be discussed later.